sbt-metals/src/main/scala/scala/meta/metals/MetalsPlugin.scala

```diff
@@ -1,17 +1,29 @@
 package metals
 
+import java.nio.file.Paths
+import java.{util => ju}
+
+import scala.io.Source
+import scala.jdk.CollectionConverters._
+import scala.util.Properties
+import scala.util.Try
+import scala.util.control.NonFatal
+
 import scala.meta.internal.sbtmetals.BuildInfo
 
-import sbt._
 import sbt.Keys
 import sbt.Keys._
+import sbt._
 import sbt.internal.inc.ScalaInstance
 import sbt.plugins.SemanticdbPlugin
 
 object MetalsPlugin extends AutoPlugin {
   override def requires = SemanticdbPlugin
   override def trigger = allRequirements
 
+  lazy val javaSemanticdbEnabled =
+    settingKey[Boolean]("Enables SemanticDB Javac plugin")
```

Comment:
Should we move this to the `autoimport` object? So that the user can deactivate it in its `build.sbt`.